### PR TITLE
Return /roles page

### LINKED ISSUE #393: Return `/roles` page
Add page for editing user roles:
- admin
- problem editor

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #393

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
https://github.com/taskany-inc/hire/assets/81121562/c748ae37-c362-46cc-9b8c-d9a8c52c857a

**Review by 9teen90nine — APPROVED** [review]

File: src/components/ProblemEditorList/ProblemEditorList.i18n/en.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "Problem Editors": "",
+    "Add Problem Editor role": "",
+    "Take the role of problem editor from": "",
+    "Add problem editor role": ""
```
Comment (nyet-ty) [inline src/components/ProblemEditorList/ProblemEditorList.i18n/en.json:5]: why this key used twice(on 3 and 5 string)? and only once in `ru.json`?

File: src/components/RolesPage/UserRolesPage.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { LayoutMain } from '../LayoutMain';
+
+import { Roles } from './Roles';
+import { tr } from './RolesPage.i18n';
+
+const UserRolesPage = () => {
+    return (
+        <LayoutMain pageTitle={tr('Users roles')}>
+            <Roles />
```
Comment (nyet-ty) [inline src/components/RolesPage/UserRolesPage.tsx:9]: why do you need to move Roles in different file?

**Review by nyet-ty — CHANGES_REQUESTED** [review]

File: src/components/ProblemEditorList/ProblemEditorList.i18n/en.json (hunk 1 above)
Comment (Katrin-kudryash) [reply src/components/ProblemEditorList/ProblemEditorList.i18n/en.json:5]: Because I need to delete an unused key

**Review by nyet-ty — APPROVED** [review]

**Pull request merged**